notebooks/00_quick_start/rlrmc_movielens.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>is rlmc added to the path for pymanopt?</p><p>once we remove that we should clean up this section and probably drop the auto reload piece?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/729/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>RLRMC is added to the python path via the command sys.path.append("../../"). I saw this being followed in the notebook ncf_movielens.ipynb.</p><p><br></p><p>pymanopt is added to the path via  the command sys.path.append("../../reco_utils/recommender/rlrmc/"). Currently, I had placed pymanopt code in this folder. We can change it if required. Please suggest the appropriate place. </p><p><br></p><p>Though pymanopt can be installed via pip, we had modified one of the files in pymanopt package  for RLRMC. </p><p><br></p>

Reply:
one question about this, is it doable to use the pip installation  of pymanopt and have `conjugate_gradient_ms.py` as a utility somewhere inside `reco_utils/recommender/rlrmc/`?

Reply:
I confirm that we can do a pip installation of pymanopt and have conjugate_gradient_ms.py as a utility somewhere inside reco_utils/recommender/rlrmc/. I have tested it in my system, after removing pymanopt from the local path (and installing pymanopt via pip install).